### DISCUSSION TITLE
Latency in Excel

### DISCUSSION THREAD

**Discussion by veronawildcats** [olivierfriard/BORIS#735] [Ideas]
Hello,
I am trying to calculate mean latency and I am using the latency function. I was wondering if there is any way I could receive the latency results in an excel sheet so I can easily calculate the mean.

**Comment by olivierfriard** [comment]
Hello,
can you send me a draft of the output do you need?
It will be useful to implement this function

**Reply by Lialulu3** [reply]
Hi, has this been implemented? I was looking for a way to do it but I didn't see anything, If it hasn't been implemented then it should be. I ended up doing it by hand since I didn't have a huge dataset.
